keda/values.yaml

```diff
@@ -3,8 +3,8 @@
 # Declare variables to be passed into your templates.
 
 image:
-  keda: docker.io/kedacore/keda:1.5.0
-  metricsAdapter: docker.io/kedacore/keda-metrics-adapter:1.5.0
+  keda: docker.io/kedacore/keda:2.0.0-alpha1
```

Comment:
Hm, why don't we have a GitHub release for this? 🤔

Reply:
Should I set this PR as a draft until v2 is released?

Reply:
I'll leave it up to @zroubalik but might be good to merge since we don't reindex the chart yet or use a v2 branch

Reply:
What about we merge this one once v2 alpha is out?

Reply:
It already is https://hub.docker.com/layers/kedacore/keda/2.0.0-alpha1/images/sha256-138b54a56b0b973597e2763e1a431a17eee11081e7a9ff07ec6eaa3085e19044?context=explore (somehow)

Reply:
I don't mean the image, but the actual release :)

Reply:
Oh ok :D